frontend/tests/jest/__mocks__/svg.tsx

```diff
@@ -0,0 +1,8 @@
+import React, { SVGProps } from 'react';
+
+const SvgrMock = React.forwardRef<SVGSVGElement, SVGProps<SVGSVGElement>>(
+   (props, ref) => <svg ref={ref} {...props} />
+);
+
+export const ReactComponent = SvgrMock;
```

Comment:
do you need both of these exports?

Reply:
Currently we don't because we use default imports in our codebase like

```ts
import logoURL from '../assets/logo.svg'
```

According to SVGR's [docs](https://react-svgr.com/docs/jest/), this `ReactComponent` export will mock imports like

```ts
import { ReactComponent as Logo } from '../assets/logo.svg'
```

So we could drop it.

Reply:
```suggestion
```